tests/app/db.py

```diff
@@ -403,6 +405,23 @@ def create_rate(start_date, value, notification_type):
     return rate
 
 
+def create_letter_rate(start_date=None, end_date=None, crown=True, sheet_count=1, rate=0.33, post_class='second'):
```

Comment:
not sure if this is the right approach - we aren't currently deleting things from letter rates in tests

Reply:
decision: this is the right approach. we should update tests to delete letterrates between runs